src/app/@ansyn/imagery/model/visualizers/visualizer-attributes.ts

```diff
@@ -0,0 +1,3 @@
+export interface IVisualizerAttributes {
+	[key: string]: string;
```

Comment:
are you sure the value is string? there is option for bool or number as i recall